### PR TITLE
Conan client test info

### PR DESCRIPTION
**Pull request by ChGen**
For #1 - Info for testing with original conan client, for reference.
Will be removed after implementing full-futured integration tests later.

### CODE REVIEW

**Comment by ChGen** [comment]
@olenagerasimova , please review.

**Review by olenagerasimova — DISMISSED** [review]
@ChGen thanks

**Comment by ChGen** [comment]
@g4s8 , I noticed that pdd github action produces an error (below), but running `pdd` ruby gem locally produces no errors.
"ERROR: conan-test/conan_server-data.tar.gz; puzzle at line #1; invalid byte sequence in UTF-8"

**Review by g4s8 — CHANGES_REQUESTED** [review]
@ChGen I'd suggest adding this files to `scr/test/resources` to use them from integration tests later. To fix `pdd` you need to exclude the directory with binary files from `pdd` tool, you can do it by updating `.pdd` file in project root with this lines:
```
--exclude path/to/dir
```
you can check this file for reference https://github.com/artipie/artipie/blob/master/.pdd

Docs: https://github.com/cqfn/pdd/#readme

**Comment by ChGen** [comment]
@g4s8 , please check, looks like all steps are fixed now.

File: README.md [hunk 1]
```diff
@@ -112,3 +112,5 @@ This is the dependency you need:
 
 ## How to configure and start Artipie Conan endpoint
 
+TODO: #1
```
Comment (g4s8) [inline README.md:115]: @ChGen what does this TODO mean?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@ChGen just a minor comment

File: README.md (hunk 1 above)
Comment (ChGen) [reply README.md:115]: Its required by pdd

Comment (g4s8) [reply README.md:115]: @ChGen it's not actually required, you can safely remove these lines

Comment (ChGen) [reply README.md]: @g4s8 , removed the line in the new commit.

**Review by g4s8 — APPROVED** [review]
@ChGen thanks

**Pull request merged**